exp/runtime/hooks/api/v1alpha1/common_types.go

```diff
@@ -71,9 +71,11 @@ type RetryResponseObject interface {
 // interface is satisfied.
 type CommonResponse struct {
 	// status of the call. One of "Success" or "Failure".
+	// +required
 	Status ResponseStatus `json:"status"`
 
 	// message is a human-readable description of the status of the call.
+	// +required
```

Comment:
```suggestion
	// +optional
```
Let's also add omitempty here

I know this is a bit strang here but:
* omitempty does not actually change anything relevant (no changes after Unmarshal + Marshal)
* this struct is not part of any CRD so we don't break any CRD by changing this
* this field is actually optional, I don't want to make it required

Reply:
Actually, it seems to be odd, but I agree with your comments 👍